src/navigator_data_ingest/base/types.py

```diff
@@ -153,3 +177,71 @@ class UnsupportedContentTypeError(Exception):
     def __init__(self, content_type: str):
         self.content_type = content_type
         super().__init__(f"Content type '{content_type}' is not supported for caching")
+
+
+@dataclass
+class Update:
+    """Class describing the results of comparing csv data against the db data to identify updates."""
+
+    db_value: Union[str, datetime]
+    csv_value: Union[str, datetime]
+    updated: bool
+    type: Literal[
```

Comment:
this looks very odd

Reply:
I assume it's something to do with the `str` subclassing?

Reply:
Nope, I can't make the simpler typing miss something....

Reply:
`class Update(BaseModel):
    """Class describing the results of comparing csv data against the db data to identify updates."""
    db_value: Union[str, datetime]
    csv_value: Union[str, datetime]
    updated: bool
    type: UpdateTypes
    field: UpdateFields`
    
    Does throw a validation error after converting to a basemodel rather than data class.

Reply:
This was left in from where I initially had Literal['FamilyDocument'...]